lisa/sut_orchestrator/libvirt/context.py

```diff
@@ -43,11 +44,16 @@ class DevicePassthroughContext:
     managed: str = ""
 
 
+@dataclass
+class GuestVmType(Enum):
+    Standard = ""
```

Comment:
Why is this an empty string? Can't we have "Standard" as the string here?

Reply:
Yes, we can have "Standard" as string.